tests/cert.json.in

```diff
@@ -0,0 +1,36 @@
+[
+    {"server_command": [@CHECKER@"openssl", "s_server", "-www", "-port", "@PORT@",
+                        "-key", "@PRIURI@", "-cert", "@CRTURI@",
+                        "-verify", "1", "-CAfile", "tests/clientX509Cert.pem"],
+     "comment": "Use ANY certificate just to ensure that server tries to authorise a client",
+     "environment": {"PYTHONPATH" : "."},
+     "server_hostname": "localhost",
+     "server_port": @PORT@,
+     "tests" : [
+       {"name" : "test-tls13-certificate-verify.py",
+        "arguments" : ["-k", "tests/clientX509Key.pem",
+                       "-c", "tests/clientX509Cert.pem",
+                       "-s", "@SIGALGS@",
+                       "-p", "@PORT@"]},
+       {"name" : "test-tls13-ecdsa-in-certificate-verify.py",
+        "arguments" : ["-k", "tests/serverECKey.pem",
+                       "-c", "tests/serverECCert.pem",
+                       "-s", "@SIGALGS@",
+                       "-p", "@PORT@"]}
+     ]
+    },
+    {"server_command": [@CHECKER@"openssl", "s_server", "-www", "-port", "@PORT@", "-key", "@ECPRIURI@", "-cert", "@ECCRTURI@"],
+     "comment": "Run test with ECDSA hostkey in pkcs11 provider",
```

Comment:
not sure if this comment is correct....

Reply:
What part of it? The `ECPRIURI` and `ECCRTURI` are the PKCS#11 URIS that the openssl server will use for TLS as far as I understand it.

Reply:
so it's using just the ECDSA key? ok, then it's fine

how do you plan to handle other key types though?

Reply:
The only other key type we have now is the EdDSA, which does not work for TLS now (under investigation) and theoretically the RSA-PSS, which does not work now in the s_server (when I created rsa-pss cert, s_server complains that the key did not match the cert -- this is also in progress/under investigation if this can be somehow resolved/worked around).
I assume when there will be more keys, I will just add separate blocks here. If there are better ways, I am all ears.

Reply:
yes, there will need to be additional blocks, it's just that this will mean additional substitutions I think....

Reply:
we have several keys (and their env vars) available)